### DISCUSSION TITLE
Font too thick?

### DISCUSSION THREAD

**Discussion by gera2342** [kovidgoyal/kitty#7350] [Q&A] [answered]
On the screenshot at the top is kitty with
```
font_family Fira Code
disable_ligatures always
```
and at the bottom is st with 
```c
static char *font = "Fira Code:pixelsize=19:antialias=true:autohint=true";
```

The "regular" font in st is more thin. I would like it to be the same in kitty. How do I achieve that?

[
![20240416_05h17m09s_grim](https://github.com/kovidgoyal/kitty/assets/6316892/19220596-797b-48ec-8b21-542ef0e9c6af)
](url)

**Comment by kovidgoyal** [comment, marked as answer]
https://sw.kovidgoyal.net/kitty/conf/#opt-kitty.text_composition_strategy

**Reply by gera2342** [reply]
Thanks!

Setting it to legacy made the font look more similar to st.
